### ISSUE TITLE
Let you create a feature and it's translation relationship in a single changeset

### ISSUE THREAD

**Issue by LegalizeAdulthood** [GrandComicsDatabase/gcd-django#575]
When migrating foreign feature names that are translation of English feature names, I have to proceed in a two-step process.  First I need to create an issue for the new feature in the foreign language.  Once that is approved, I have to create another issue to mark the translation relationship between the two features.  It would be much simpler if we could create the translated feature as a single changeset.

**Comment by LegalizeAdulthood** [comment]
It also doesn't help that the "new features" are listed at the top of my editing page, but the "feature relationship" is listed at the bottom of my editing page...

**Comment by jochengcd** [comment]
We have generally no setup for combining different revisions into one. Issues/stories are specially handled.

Nice to have, but first we would need a full overhaul of the oi, not just for this.
